tests/unittest/llmapi/test_reasoning_parser.py

```
@@ -88,6 +89,48 @@ def test_deepseek_v4_reasoning_parser_streams_when_thinking():
+
+    assert result.content == "visible"
+    assert result.reasoning_content == "hidden"
+
+
+@pytest.mark.parametrize("chat_template_kwargs", [{"enable_thinking": False}])
+def test_poolside_v1_reasoning_parser_extracts_when_not_thinking(
+        chat_template_kwargs: dict):
+    reasoning_parser = ReasoningParserFactory.create_reasoning_parser(
+        "poolside_v1", chat_template_kwargs)
+
+    result = reasoning_parser.parse(f"visible")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)